tests/python_tests/test_preemption.py

```diff
@@ -22,7 +22,6 @@ def get_beam_search_seq_len_300() -> GenerationConfig:
     generation_config.num_beam_groups = 3
     generation_config.num_beams = 6
     generation_config.max_new_tokens = 300
-    generation_config.num_return_sequences = 3
```

Comment:
Is this change needed?

Reply:
It's just a cleanup. See, that we assign `num_return_sequences` below, so this part was overrode anyway.